spec.yaml

```diff
@@ -218,6 +231,66 @@ definitions:
           (e.g. no use of special characters) providing more information
           on the cause of the error.
         example: RESOURCE_NOT_FOUND
+  Discovery:
+    type: object
+    required:
+      - enabled
+      - apiVersion
+      - endpoint
+      - resourceTypes
+    properties:
+      enabled:
+        type: boolean
+        description: Whether the OCM service is enabled at this endpoint
+      apiVersion:
+        type: string
+        description: The OCM API version this endpoint supports
+        example: 2.0alpha
+      endpoint:
+        type: string
+        description: The URI of the OCM API available at this endpoint
+        example: https://my-cloud-storage.org/ocm
+      provider:
+        type: string
+        description: A friendly branding name of this endpoint
+        example: MyCloudStorage
```

Comment:
I noticed this extra property which makes sense and it's not required so it checks out.